src/ui/components/NavBar.js

```diff
@@ -0,0 +1,39 @@
+import { testStep } from "../../common/helpers/pwHelpers.js";
+
+export class NavBar {
```

Comment:
It is better not to abbreviate names in classes, but to use full names, for example "NavigationBar"